genon/preprocessor/tests/smoke/test_pdf_convert_smoke.py

```
@@ -0,0 +1,43 @@
+            PPT, id="ppt",
+            marks=pytest.mark.skipif(not PPT.exists(), reason="ppt_sample.ppt not found"),
+        ),
+        pytest.param(
+            PPTX, id="pptx",
+            marks=pytest.mark.skipif(not PPTX.exists(), reason="pptx_sample.pptx not found"),
+        ),
+    ],
+)
+@pytest.mark.parametrize("use_pdf_sdk", [True, False], ids=["sdk", "libreoffice"])
+async def test_pdf_convert_smoke(basic_processor, sample: Path, use_pdf_sdk: bool):
+    """입력 ext × {PDF SDK, LibreOffice} 모두 vectors 반환 확인."""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)